src/test/java/software/amazon/cloudformation/resource/ResourceTypeSchemaTest.java

```diff
@@ -121,6 +124,7 @@ public void getSingletonReplacementStrategy() {
 
         String replacementStrategy = schema.getReplacementStrategy();
         assertThat(replacementStrategy).isEqualTo("delete_then_create");
+        assertTrue(schema.getTaggable());
```

Comment:
nit: 

can we use a pattern like assertThat().isTrue()

Reply:
Sure. Thanks.